### PR TITLE
typo and legacy sql server syntax

### PR DESCRIPTION
**Pull request by labkey-matthewb**
#### Rationale
fix typo

#### Related Pull Requests
- 

#### Changes
-

### CODE REVIEW

**Review by labkey-susanh — CHANGES_REQUESTED** [review]
Also need to modify line 1244 to use `expm.cpastype` instead of just `cpastype`

File: experiment/src/org/labkey/experiment/api/ExpMaterialTableImpl.java [hunk 1]
```diff
@@ -1229,25 +1229,32 @@ void executeIncrementalRollup()
...
+                incremental
+                    .append("UPDATE st\n")
+                    .append("SET aliquotcount = expm.aliquotcount, availablealiquotcount = expm.availablealiquotcount, aliquotvolume = expm.aliquotvolume, availablealiquotvolume = expm.availablealiquotvolume, aliquotunit = expm.aliquotunit\n")
+                    .append("FROM temp.${NAME} st, exp.Material expm\n")
+                    .append("WHERE expm.rowid = st.rowid AND expm.cpastype = ").appendValue(_lsid,d).append(" AND (\n")
+                    .append("    COALESCE(st.aliquotcount,-2147483648) <> COALESCE(expm.aliquotcount,-2147483648) OR ")
+                    .append("    COALESCE(st.availablealiquotcount,-2147483648) <> COALESCE(expm.availablealiquotcount,-2147483648) OR ")
+                    .append("    COALESCE(st.aliquotvolume,-2147483648) <> COALESCE(expm.aliquotvolume,-2147483648) OR ")
+                    .append("    COALESCE(st.availablealiquotvolume,-2147483648) <> COALESCE(expm.availablealiquotvolume,-2147483648) OR ")
+                    .append("    COALESCE(st.aliquotunit,-2147483648) <> COALESCE(expm.aliquotunit,-2147483648)")
```
Comment (labkey-susanh) [inline experiment/src/org/labkey/experiment/api/ExpMaterialTableImpl.java]: AliquotUnit is a string, not a number. Will this `COALESCE` work well?

Comment (labkey-matthewb) [reply experiment/src/org/labkey/experiment/api/ExpMaterialTableImpl.java]: Seems to work fine. I can change to something else.

Comment (labkey-susanh) [reply experiment/src/org/labkey/experiment/api/ExpMaterialTableImpl.java]: Could probably just be the empty string.

**Review by labkey-susanh — APPROVED** [review]

**Pull request merged**